### PR TITLE
Add back Apache 2 on 2 intances article

### PR DESCRIPTION
**Pull request by kylelaffoon**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-146f8d677c/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-cce4306d5b/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-ad07e42a45/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ca7192ce0e/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-819338050d/how-to/)

File: content/cloud-networks/set-up-apache2-web-server-on-two-instances-with-shared-ip.md [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+            "adminPass": "q5dmoSkbYLF6"
+         }
+        }
+
+4. Get the details for the slave server and confirm that slave server is in
+   the same publicIPZoneId as master server.
+
+   *Request:*
+   
+      curl -s https://dfw.servers.api.rackspacecloud.com/v2/5831008/servers/ffec9d55-2d54-4718-bc3a-0d47fb8c52c1
```
Comment (stephamon) [inline content/cloud-networks/set-up-apache2-web-server-on-two-instances-with-shared-ip.md]: This code needs another space or two. It's not rendering as a code block in the preview

File: content/cloud-networks/set-up-apache2-web-server-on-two-instances-with-shared-ip.md [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+   *File contents:*
+   
+        master-instance-hostname 10.23.233.113/24
+
+10. (Slave) Create the ***/etc/heartbeat/ha.cf*** file on the slave server and
+   enter the following text. The lines with comments are the ones that have to be modified.
+   
+   *File contents:*
+   
+        logfacility daemon
```
Comment (kylelaffoon) [inline content/cloud-networks/set-up-apache2-web-server-on-two-instances-with-shared-ip.md]: Thanks @stephamon. Applied your suggestion.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5970ace100/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-17dc900212/how-to/)

**Comment by catlook** [comment]
wow...what a painful PR...thanks, y'all!!!

**Pull request merged**